src/main/java/seedu/address/model/person/Person.java

```diff
@@ -13,45 +13,27 @@
  * Represents a Person in the address book.
  * Guarantees: details are present and not null, field values are validated, immutable.
  */
-public class Person {
+public class Person extends Task {
 
     // Identity fields
-    private final Name name;
-    private final Phone phone;
     private final Email email;
 
     // Data fields
-    private final Address address;
     private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
     public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, address, tags);
-        this.name = name;
-        this.phone = phone;
+        super(name, phone, address);
+        requireAllNonNull(address, tags);
```

Comment:
address should be module code i think!

Reply:
address name should change to module code after merging the PR that changed the address field to module code